.github/workflows/secrets-sdk.yml

```diff
@@ -67,13 +66,7 @@ jobs:
               CARGO=cross npm run build -- --target x86_64-unknown-linux-musl
           - host: macos-latest
             target: aarch64-apple-darwin
-            build: |
-              sudo rm -Rf /Library/Developer/CommandLineTools/SDKs/*;
-              export CC=$(xcrun -f clang);
-              export CXX=$(xcrun -f clang++);
-              SYSROOT=$(xcrun --sdk macosx --show-sdk-path);
-              export CFLAGS="-isysroot $SYSROOT -isystem $SYSROOT";
-              npm run build -- --target aarch64-apple-darwin
```

Comment:
Its not convenient that the pipeline broke, but at least we were able to remove these steps from the aarch64 build 😋